### DISCUSSION TITLE
碰到一个地方的了！

### DISCUSSION THREAD

**Discussion by AlexShyXie** [ShixiangWang/self-study#90] [灌水]
大哥你好~，我是SYSUCC影像科的，在做深度学习和生信，求交流

**Comment by ShixiangWang** [comment]
可以的。有什么问题都可以在这里讨论。或者邮箱 w_shixiang@163.com for private communication.

**Reply by AlexShyXie** [reply]
大佬，想问一下，现在肿瘤生信都有哪些方向可以做，我看到的大多数都是灌水发低分文章，如果想发高分文章，应该怎么学呢，是需要用linux开发新的计算工具吗？还是利用GEO和TCGA进行全程用R也能可以？

**Reply by ShixiangWang** [reply]
我自己总结大概有下面几个方向。如果你在图像方向可能还可以做临床病理检测和相关AI技术开发之类的。核心分析是纯生信以及干湿结合。能发多好的期刊文章取决于课题研究的创新性、研究结果创新性和完备性、学术写作（展示）水平。

- 以生信/湿实验为手段的Pure探索性/系统性科学研究
- 基于新的数据采集技术的方法和流程开发，例如基因组比对，变异检测，表观遗传修饰鉴定等
- 针对性需求的方法与工具开发
- 功能数据库的整合与挖掘，工具开发。例如TCGA一系列挖掘和工具文章
- 基础实验启发式的数据分析辅助研究
- 数据库开发与维护

生信研究的核心基础有3个方面：

- 生物学研究背景
- 数理统计
- 计算分析（编程）

根据自己的研究兴趣和课题去学习以上所需的三部分即可。发高分文章没有怎么学一说，目前的高分要么创新够强，要么堆的资源够多，你提的两点属于上面的第3点基础，跟文章能发多好没有必然联系，只是一些研究内容所需要的基础。

**Reply by AlexShyXie** [reply]
醍醐灌顶，太感谢了

**Comment by AlexShyXie** [comment]
大佬，我想咨询一下，RNAseq的差异分析貌似需要用read counts来做，不能用FPKM，但是我又听说TPM也可以用来做差异分析，不知道这个说法对不对

**Reply by ShixiangWang** [reply]
你需要看你的应用场景和使用的工具。当前流行的工具一般是以counts作为输入（负二项分布的数据）。像FPKM或TPM可以log化后（正态化的数据）走芯片分析类似的流程。

**Reply by AlexShyXie** [reply]
> 你需要看你的应用场景和使用的工具。当前流行的工具一般是以counts作为输入（负二项分布的数据）。像FPKM或TPM可以log化后（正态化的数据）走芯片分析类似的流程。

原来如此，太感谢了
